packages/pathfinder/src/pathfinder.ts

```diff
@@ -0,0 +1,370 @@
+const config =  require('../../../config.json');
+const { Pool } =  require('pg');
+const pool = new Pool({
+    user: 'postgres',
+    host: 'interstellar.market',
+    database: 'silent_shop',
+    password: config.pg.password,
+    port: 5432,
+});
+
+const StellarSdk = require('stellar-sdk'); 
+const server = new StellarSdk.Server('https://horizon.stellar.org');
```

Comment:
It would be ideal to be able to set this up in the constructor.

Reply:
pulled in server and left SDK as es6 module